### PR TITLE
PieChart: update Pies with Equal Sizes demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/PiesWithEqualSize/Angular/description.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can use an `*ngFor` notation to iterate through every PieChart option.
+
+When you display PieChart components side by side, their pies may differ in size. To eliminate these differences, you can collect all charts in a single size group. To do this, set the charts [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property to identical values.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/PiesWithEqualSize/Angular/description.md]: A == single already. If you say "a single", you put a special emphasis on the aspect of singularity. Do you really need this? I have my doubts.

Set to identical values? I understand what you mean, yet it sounds somewhat strange. All you actually need to do is to set the sizeGroup property of every component to the same value no matter what it is. 0, for example. Or 2. Or whatever...

```suggestion
When you display PieChart components side by side, their pies may differ in size. To make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```

File: JSDemos/Demos/Charts/PiesWithEqualSize/React/description.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can use the [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) method to iterate through every PieChart option.
+
+When you display PieChart components side by side, their pies may differ in size. To eliminate these differences, you can collect all charts in a single size group. To do this, set the charts [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property to identical values.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/PiesWithEqualSize/React/description.md]:
```suggestion
When you display PieChart components side by side, their pies may differ in size. To make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```

File: JSDemos/Demos/Charts/PiesWithEqualSize/Vue/description.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can use a `v-for` notation to iterate through every PieChart option.
+
+When you display PieChart components side by side, their pies may differ in size. To eliminate these differences, you can collect all charts in a single size group. To do this, set the charts [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property to identical values.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/PiesWithEqualSize/Vue/description.md]:
```suggestion
When you display PieChart components side by side, their pies may differ in size. To make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```

File: JSDemos/Demos/Charts/PiesWithEqualSize/description.md [hunk 4]
```diff
@@ -1 +1,3 @@
-When several PieChart components are displayed side by&nbsp;side, their pies may differ in&nbsp;size. To&nbsp;eliminate these differences, you can collect all **PieCharts** in&nbsp;a&nbsp;single size group by&nbsp;setting their **sizeGroup** property to&nbsp;identical values.
\ No newline at end of file
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can declare an object with options and use this object in every PieChart.
+
+When you display PieChart components side by side, their pies may differ in size. To eliminate these differences, you can collect all charts in a single size group. To do this, set the charts [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property to identical values.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/PiesWithEqualSize/description.md]:
```suggestion
When you display PieChart components side by side, their pies may differ in size. make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]
This PR has the same issue copied across four similar topics. Please stick to simplicity of your language. The less psychic energy your reader needs to parse your text, the better.

File: JSDemos/Demos/Charts/PiesWithEqualSize/Angular/description.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can use an `*ngFor` notation to iterate through every PieChart option.
+
+When you display PieChart components side by side, their pies may differ in size. To make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```
Comment (arminal) [inline JSDemos/Demos/Charts/PiesWithEqualSize/Angular/description.md]:
```suggestion
When you display PieChart components side by side, their pies [do you mean the pies that belong to these components?] may differ in size. You join all charts into a size group to resize these pies to match each other. To do so, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```
Apply this edit to other similarly-phrased sections.

File: JSDemos/Demos/Charts/PiesWithEqualSize/description.md [hunk 6]
```diff
@@ -1 +1,3 @@
-When several PieChart components are displayed side by&nbsp;side, their pies may differ in&nbsp;size. To&nbsp;eliminate these differences, you can collect all **PieCharts** in&nbsp;a&nbsp;single size group by&nbsp;setting their **sizeGroup** property to&nbsp;identical values.
\ No newline at end of file
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can declare an object with options and use this object in every PieChart.
+
+When you display PieChart components side by side, their pies may differ in size. make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```
Comment (arminal) [inline JSDemos/Demos/Charts/PiesWithEqualSize/description.md]: Several errors here. Use my notes from above to rewrite this section and others like it.

**Pull request merged**